### ISSUE TITLE
`node/_tools/setup.ts` doesn't work since Deno v1.13.0

### ISSUE THREAD

**Issue by kt3k** [denoland/deno_std#1298]
The downloaded file has a wrong length with Deno >= v1.13.0, and it cannot be decompressed. The tool doesn't have a problem with Deno <= v1.12.2.

**Comment by kt3k** [comment]
Decompression part was fixed by #1299, but the downloading part still has a problem

**Comment by kt3k** [comment]
The following script illustrates the problem of setup.ts:

```ts
// Writes 100KB text file
await Deno.writeTextFile("dummy.txt", "123456789\n".repeat(10_000));
// Serves the above file at localhost
const p = Deno.run({
  cmd: [
    Deno.execPath(),
    "run",
    "-A",
    "https://deno.land/std/http/file_server.ts",
  ]
});
await new Promise((resolve) => setTimeout(resolve, 300));

// Fetches the above file and saves it (copy0.txt). (uses Deno.write in for loop)
const res0 = await fetch("http://localhost:4507/dummy.txt");
const file0 = await Deno.open("copy0.txt", { create: true, truncate: true, write: true });
for await (const chunk of res0.body!) {
  await Deno.write(file0.rid, chunk);
}
file0.close();

// Fetches the above file and saves it (copy1.txt). (uses Deno.writeSync in for loop)
const res1 = await fetch("http://localhost:4507/dummy.txt");
const file1 = await Deno.open("copy1.txt", { create: true, truncate: true, write: true });
for await (const chunk of res1.body!) {
  Deno.writeSync(file1.rid, chunk);
}
file1.close();

p.close();

console.log("copy0.txt", Deno.readTextFileSync("copy0.txt").length);
console.log("copy1.txt", Deno.readTextFileSync("copy1.txt").length);
```

The above script executes like the below:

```shellsession
$ deno run -A repro.ts
Check file:///Users/kt3k/denoland/deno_std/repro.ts
HTTP server listening on http://localhost:4507/
[2021-10-17 17:48:01] "GET http://localhost:4507/dummy.txt" 200
[2021-10-17 17:48:01] "GET http://localhost:4507/dummy.txt" 200
copy0.txt 99232
copy1.txt 100000
```

(The file size of the first file is 768 bytes smaller than the correct file size.)

The problem is that if we use `for await (const ... of response.body) { await Deno.write(...) }` structure, then the downloaded file lost some bytes (here 768 bytes). On the other hand, if we change `await Deno.write` part to `Deno.writeSync`, then the problem disappears.

Side note: The amount of lost bytes are always multiple of `256` from my observation.
